charts/sck-otel/templates/_config.tpl

```diff
@@ -368,4 +376,22 @@ service:
         {{- end }}
         {{- end }}
     {{- end }}
+    {{- if .Values.journaldLogs.enabled }}
+    logs/journald:
+      receivers:
+        - journald
+      processors:
+        - batch
+        - resource/splunk
+      exporters:
+        {{- if .Values.splunkPlatform.endpoint }}
+        - splunk_hec/platform
+        {{- end }}
+        {{- if .Values.splunkObservability.logsEnabled }}
+        {{- if or .Values.splunkObservability.ingestUrl .Values.splunkObservability.realm }}
```

Comment:
you can use this one here as well `{{- if eq (include "splunk-otel-collector.sendLogsToO11y" .) "true" }}`